spec/support/factory_girl.rb

```diff
@@ -9,4 +9,9 @@
     password_confirmation 'password'
   end
 
+  factory :bucket_list do
+    sequence(:title) {|n| "My List ##{n}" }
+    description 'I go places'
+    is_public 'false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.